### PR TITLE
feat: add jwt token signing and verification logic

### PR DESCRIPTION
**Pull request by shwetha-manvinkurke**
Add [access token](https://www.twilio.com/docs/iam/access-tokens) generation functionality

### CODE REVIEW

**Review by eshanholtz — COMMENTED** [review]
Did you run coverage analysis? If so, will you post analysis results?

File: README.md [hunk 1]
```diff
@@ -495,6 +495,69 @@ func main() {
...
+    Incoming: jwt.Incoming{Allow: true},
+    Outgoing: jwt.Outgoing{
+        ApplicationSid:    applicationSid,
+        ApplicationParams: "",
+    },
+}
+
+jwtToken.AddGrant(voiceGrant)
+token, _ := jwtToken.ToJwt()
+fmt.Println(token)
```
[the remark this answers is not in the document]
Comment (shwetha-manvinkurke) [reply README.md:535]: > Did you run coverage analysis? If so, will you post analysis results?

Here's the coverage gist:
```
ok      github.com/twilio/twilio-go/client/jwt  0.441s  coverage: 92.7% of statements
ok      github.com/twilio/twilio-go/client/jwt/taskrouter       0.525s  coverage: 91.4% of statements
ok      github.com/twilio/twilio-go/client/jwt/util     0.356s  coverage: 100.0% of statements
```
[coverage.out.txt](https://github.com/twilio/twilio-go/files/7137187/coverage.out.txt)

[the remark this answers is not in the document]
Comment (shwetha-manvinkurke) [reply README.md:535]: > are the printlns necessary?

not really. Removed it.

**Review by eshanholtz — APPROVED** [review]
🎉

**Review by JenniferMah — APPROVED** [review]

**Pull request merged**